events/imperia_discord_break.txt

```diff
@@ -2739,6 +2759,11 @@ discord_break.1 = {
 			}
 		}
 		every_scope_building = {
+			OR = {
+				exists = scope:major_strike_building_1
+				exists = scope:major_strike_building_2
+				exists = scope:major_strike_building_3
+			}
```

Comment:
whoops this one is fine. If they aren't defined then simply no buildings get the modifier. There should be error spam though, to prevent error spam this should be an and.